roles/confluent.kafka_broker/defaults/main.yml

```diff
@@ -33,6 +33,17 @@ kafka_broker_sysctl_file: /etc/sysctl.conf
 
 kafka_broker_default_interal_replication_factor: "{{ [ groups['kafka_broker'] | length, 3 ] | min }}"
 
+kafka_broker_metrics:
+  reporters: []
```

Comment:
if i follow your code the default is to have no reporters now, when before it was
metric.reporters: io.confluent.metrics.reporter.ConfluentMetricsReporter

Reply:
The templates are taking care of including the ConfluentMetricsReporter and ConfluentMetricsInterceptor(s)

Reply:
I've made a few changes to clean this up